internal/agent/disk_utils.go

```
@@ -73,11 +82,131 @@ func (a *Agent) shouldIncludeDisk(mountpoint, device, fstype string) bool {
 		fstype == "cgroup2" {
-		return false
+		return false, false
 	}
 
 	// Include everything else by default
-	return true
+	return true, false
+}
+
+// shouldIncludeDisk determines if a disk should be included in monitoring based on filter rules
+func (a *Agent) shouldIncludeDisk(mountpoint, device, fstype string) bool {
```

golangci-lint (2.11.4)

[error] func (*Agent).shouldIncludeDisk is unused

(unused)